tests/integration_tests/data_steward/retraction/retract_utils_test.py

```diff
@@ -0,0 +1,50 @@
+# Python imports
+import os
+import time
+from unittest import TestCase
+
+# Third party imports
+import google.cloud.bigquery as gbq
+
+# Project imports
+import app_identity
+import retraction.retract_utils as ru
+
+
+class RetractUtilsTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = app_identity.get_application_id()
+        self.dataset_id = os.environ.get('UNIONED_DATASET_ID')
+        self.client = gbq.Client()
+
+    def test_is_labeled_deid(self):
+        dataset = self.client.get_dataset(
+            f'{self.project_id}.{self.dataset_id}')
+        dataset.labels = {"de_identified": "true"}
+        self.client.update_dataset(dataset, ["labels"])
+        actual = ru.is_labeled_deid(self.client, self.project_id,
+                                    self.dataset_id)
+        self.assertTrue(actual)
+
+        dataset = self.client.get_dataset(
+            f'{self.project_id}.{self.dataset_id}')
+        dataset.labels = {"de_identified": "false"}
+        self.client.update_dataset(dataset, ["labels"])
+        actual = ru.is_labeled_deid(self.client, self.project_id,
+                                    self.dataset_id)
+        self.assertFalse(actual)
+
+        dataset = self.client.get_dataset(
+            f'{self.project_id}.{self.dataset_id}')
+        dataset.labels = {"de_identified": None}
+        self.client.update_dataset(dataset, ["labels"])
+        actual = ru.is_labeled_deid(self.client, self.project_id,
+                                    self.dataset_id)
+        self.assertIsNone(actual)
```

Comment:
This class needs a `tearDown(self)` function.  The test dataset is still hanging around after the test is complete.  The test should clean up after itself.

Reply:
This test is not creating the dataset, but rather reusing one of the default test datasets that already exist. The labels that are assigned as part of the tests are removed by the end of the test.